modules/javafx.web/src/test/java/test/javafx/scene/web/LocalStorageTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2011, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright should be a single year (2022)

Reply:
done